### DISCUSSION TITLE
Thunderbolt 3 (Hotplug)/ eGPU/ Docks/ and related

### DISCUSSION THREAD

**Discussion by tylernguyen** [tylernguyen/x1c6-hackintosh#136] [General]
Please use this thread for discussions for the aforementioned topics.

Also, please refrain from creating an issue thread unless you have a specific and demonstrable issue. 

Best,
-Tyler

**Comment by Charlyo** [comment]
Regarding [NTFY talk](https://github.com/tylernguyen/x1c6-hackintosh/issues/127#issuecomment-855249111) @kushwavez "If no drivers are loaded" appears to the Thunderbolt section, this mean you have lost Thunderbolt Bus and Thunderbolt Local Node as a normal mac should behave. The connection at that moment is still Pci-2-pci bridge

**Reply by kushwavez** [reply]
What is that mean in practice? 
As far as I can see there is no differences in usage.

But of course it would be better to have drivers loaded, but then the wake process is slow, and the hot-plug starts not working randomly

**Reply by Charlyo** [reply]
This means loosing:
- The being able to connect your laptop directly with another mac in target disk mode to directly access its drive to recover / move stuff around.
- Connect directly with another mac and sharing internet connection between them.
- More stuff that I do not know.

I am also facing hot-plug issues with my T480. Long story short, I adapted the SSDTS @benbender made to my T480 since the majority of APCI related tables are very similar in both computers (x1c6 vs T480). Thunderbolt works prior to sleep, but cannot get hot-plug after sleep as well.

Also, I think you also need another rename: From INI to OINI and a sleep ssdt. Have you included that @kushwavez ?

**Comment by velaar** [comment]
You are also losing native power management. You should see ~3-4W of extra drain because of TBT controller staying powered up. And CPU will fail to enter states below C3.
